ddtrace/bootstrap/sitecustomize.py

```diff
@@ -43,17 +43,23 @@
 
 
 def cleanup_loaded_modules():
+    from ddtrace.settings.asm import config as asm_config
```

Comment:
we cannot import product-specific stuff in this module or we risk circular imports